docker-compose.yml

```diff
@@ -0,0 +1,85 @@
+services:
+  dittofs:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    container_name: dittofs-server
+    image: dittofs:latest
+    ports:
+      - "12049:12049/tcp"
+      - "2049:2049/udp"
```

Comment:
This port should be aligned with the TCP one. At the moment, we don't support UDP yet though, so we can safely remove this mapping